logic/src/commonMain/kotlin/com/wire/kalium/logic/data/asset/Asset.kt

```diff
@@ -57,3 +70,26 @@ fun isAudioMimeType(mimeType: String): Boolean = mimeType in setOf(
 fun isVideoMimeType(mimeType: String): Boolean = mimeType in setOf(
     "video/mp4", "video/webm", "video/3gpp", "video/mkv"
 )
+
+fun getExtensionFromMimeType(mimeType: String?): String? {
+    return mimeType?.let { mimeTypeToExtensionMap[it.lowercase()] }
```

Comment:
I'm wondering whether it wouldn't be more effective to simply keep the last part of the mimeType, i.e. the second part after the `/jpg` since they match 99.9% of the times. Otherwise we are limiting ourselves to this small list. Zip or pdf files will not be able to be mapped. 
In general I would say we should rely on obtaining the extension file via `Uri.getMimeType()` method in the app utils folder as it's pretty reliable and supported by the Android system.

If that doesn't work because we want to centralise this logic in Kalium or we can't have access to the app `Context` and we still want to go for a list of known mimeTypes I'd suggest to expand it with more known formats also for `application/xxx` types. The official IANA list can be found [here](https://www.iana.org/assignments/media-types/media-types.xhtml).

Reply:
You're right, I also talked with Kubaz about it. I created separate implementations for different platforms - on Android we use `android.webkit.MimeTypeMap`, on jvm I added `org.apache.tika.mime.MimeTypes` which is recommended, other platforms still use previous implementation, we can update the implementation for iOS or JS if needed.